### DISCUSSION TITLE
Subscription not recognized

### DISCUSSION THREAD

**Discussion by Ciutnik** [devemux86/cruiser#359] [Q&A] [answered]
A strange thing happened today. On one device the subscription (annual) was not recognized, while on another it is obviously valid until May 2024. Of course, both devices are logged in to the same Google account.

Resetting or clearing the cache didn't help. I'm still being offered a new subscription starting today. It's definitely some bug on my phone, but maybe there is some way other than reinstalling the application?

**Comment by devemux86** [comment]
@Ciutnik thanks for the report.

App subscriptions are validated through the Google Play app, usually each time internet is available.

If you go to your subscriptions in Google Play app, do you see them?

> Resetting or clearing the cache didn't help. 

Cruiser or Google Play cache?

Can you clear the cache of Google Play app, login again in Google Play, check your subscriptions in Google Play app,
and then start Cruiser again? (also restart the phone)

**Reply by Ciutnik** [reply, marked as answer]
I kept seeing an active subscription in the Play Store. I cleared the cache of both the Play Store and the application, but it didn't help. So I cleared ALL Play Store data. After restarting the phone again, everything started working! 👍

Thanks for fast reaction. The alarm has been canceled, maybe someone will find the above information useful in the future 🙂

**Reply by devemux86** [reply]
@Ciutnik very nice that the problem was solved! :+1: 

As the subscriptions are cached locally by the Google Play app,
the recommended solution is usually to clear the Google Play app's cache / data.

**Reply by Ciutnik** [reply]
Yes, I will remember this in the future 🙂

Today we have "All Souls' Day" here, and on this occasion I wanted to take advantage of walking navigation through the huge cemetery. I had several graves of family and friends to visit, and this was such an unpleasant surprise... 😉 

Nothing bad happened because follow mode was still working. I reached every place precisely 👍

**Comment by devemux86** [comment]
Also happened here, Google Play suddenly didn't recognize any of my subscriptions in all apps.

It worked again when I opened the Subscriptions page of Google Play app with active internet,
so that it finds the subscriptions again (like in the images).

Another idea as [mentioned above](https://github.com/devemux86/cruiser/discussions/359#discussioncomment-7447183) is to clear the data / cache of the Google Play app.

<img src="https://github.com/devemux86/cruiser/assets/3484020/83093811-6810-4c7c-a44b-7ddd9ceafd72" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/87e7c53e-9fc0-4a8c-aa78-fceb5844a3cc" width="50%"/>
